js/containers/selection-checkbox.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import { setQuestionSelected } from '../actions'
+import SelectionCheckBox from '../components/selection-checkbox'
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    onSelectChange: (key, selected) => {
+      dispatch(setQuestionSelected(key, selected))
+    }
```

Comment:
I've been writing actions mapping in the same way, although I think it can be simplified if the action has the same interface as the final property. I think this should work too:
```javascript
const mapDispatchToProps = { setQuestionSelected }
```
or even:
```javascript
import * as actions from '../actions'
(...)
connect(mapStateToProps, actions)(SelectionCheckBox)
```
it's popular pattern, but that will pass all the actions to the container (which is probably okay too).
It's not worth changing now, but we can start using simpler pattern in the future.